include/gridtools/stencil/frontend/expandable_run.hpp

```diff
@@ -157,6 +168,11 @@ namespace gridtools {
                     expanded_run<1, spec_t, Backend, Is...>(grid, offset, fields...);
             }
 
+            template <size_t, class... Ts>
+            void run_impl(Ts...) {
+                static_assert(sizeof...(Ts) < 0, "Unexpected gridtools::stencil::expandable_run first argument.");
```

Comment:
```suggestion
                static_assert(sizeof...(Ts) < 0, "Unexpected first argument of gridtools::stencil::expandable_run: the first argument should be the expand factor (integer).");
```

Reply:
Nope. Factor is a compile time parameter, the first runtime argument that is meant here is a spec.